### PR TITLE
Auto feed sniping spots from http://pkmngotrading.com/wiki/Nests

### PR DESCRIPTION
**Pull request by kivelaBE**
capture
This update contains source files: just copying them into your folder will not work !

Created an update that will fetch the spots from http://pkmngotrading.com/wiki/Nests as sniping spots, therefor no longer requiring manual feeding of spots.

You need to clear out all the locations in your config file like below to enable the auto detect of locations from the webfeed:

"PokemonToSnipe": {
    "Locations": [], 
    "Pokemon": [
      "abra",
      "bellsprout", ... ] -> keep the part of the pokemon as is !

In the future website should be changed to a bot fed actual list.

### CODE REVIEW

File: PoGo.NecroBot.Logic/Tasks/SnipePokemonTask.cs [hunk 1]
```diff
@@ -48,7 +49,7 @@ public override int GetHashCode()
 
         public override string ToString()
         {
-            return latitude.ToString("0.0000") + ", " + longitude.ToString("0.0000");
+            return latitude.ToString("0.0000") + ", " + latitude.ToString("0.0000");
```
Comment (zenwaichi) [inline PoGo.NecroBot.Logic/Tasks/SnipePokemonTask.cs]: Is this a mistake?

**Comment by Newgamer1** [comment]
Sniping don´t work. I Changed my PC-settings to USA, also de "." and "," thing.
Do you have any solution?

File: PoGo.NecroBot.Logic/Tasks/SnipePokemonTask.cs (hunk 1 above)
Comment (kivelaBE) [inline PoGo.NecroBot.Logic/Tasks/SnipePokemonTask.cs]: I didn't change that part, but will change, thnx !

**Comment by Sureiya** [comment]
I don't think its really safe to pull data from a public wiki.

**Comment by kivelaBE** [comment]
@Newgamer1  what message do you receive ?

**Comment by randysoria** [comment]
Same here the Sniping dosent work anymore, please do u have any solution for this problem please ill be looking around for a solution.
TY

**Comment by kivelaBE** [comment]
@Sureiya : true, but this is just a proof-implementation.  Ideally their sniping bot would deliver a more steadfast and trustworthy source

**Comment by kivelaBE** [comment]
@randysoria : what did you do to test this implementation?

**Comment by Newgamer1** [comment]
@kivelaBE 

I Get this message:
[19:52:51] (INFO) [Sniper] Scanning for Snipeable Pokemon at 38.5568074864611,-121.238379478455...
[19:52:51] (INFO) [Sniper] No Pokemon found to snipe!
[19:52:51] (INFO) [Sniper] Scanning for Snipeable Pokemon at -33.859019,151.213098...
[19:52:51] (INFO) [Sniper] No Pokemon found to snipe!
[19:52:51] (INFO) [Sniper] Scanning for Snipeable Pokemon at 47.5014969,-122.0959568...
[19:52:52] (INFO) [Sniper] No Pokemon found to snipe!
[19:52:52] (INFO) [Sniper] Scanning for Snipeable Pokemon at 51.5025343,-0.2055027...
[19:52:52] (INFO) [Sniper] No Pokemon found to snipe!

My settings:
http://prntscr.com/bzba48

2 hours ago, it worked really good ;(

**Comment by randysoria** [comment]
Nothing, it was working good, I put coords close to my home and the sniping Coords the same, but it said (No Pokemon Found to Snipe!)

**Comment by randysoria** [comment]
Same to me @kivelaBE !!

**Comment by randysoria** [comment]
(INFO) [Sniper] Scanning for Snipeable Pokemon at 38.5568074864611,-121.238379478455...
[21:50:04] (INFO) [Sniper] No Pokemon found to snipe!
[21:50:04] (INFO) [Sniper] Scanning for Snipeable Pokemon at -33.859019,151.213098...
[21:50:04] (INFO) [Sniper] No Pokemon found to snipe!
[21:50:04] (INFO) [Sniper] Scanning for Snipeable Pokemon at 47.5014969,-122.0959568...
[21:50:04] (INFO) [Sniper] No Pokemon found to snipe!
[21:50:04] (INFO) [Sniper] Scanning for Snipeable Pokemon at 51.5025343,-0.2055027...
[21:50:04] (INFO) [Sniper] No Pokemon found to snipe!

**Comment by kivelaBE** [comment]
Did any of you actually copy these files and recompile ?

**Comment by randysoria** [comment]
No i didnt copy anything, the Sniping just stop working..

**Comment by Newgamer1** [comment]
No I didn´t.

**Comment by zenwaichi** [comment]
So I'm not the only who hasn't been able to snipe in the last hours. 
@kivelaBE  nope I've started today and only tried the executable, not checked source yet

**Comment by kivelaBE** [comment]
The sniping got wrecked a few hours ago due to an update of pokevision.com, this is a development pull in which I incorporated a fix made by another developer and added the auto-feeding of >450 sniping locations.  So it will only work after you recompile from source with these changes.

**Comment by kaeus** [comment]
@kivelaBE I like the idea, but it needs to be something optional IMO. Personally, I prefer to either watch a feed for rares or snipe specific spots at a time, rather than spam all the nests.

To everyone else, if you are not commenting on the PR specifically, please stop spamming the damn thread.

**Comment by nail222** [comment]
@kivelaBE I have been gettings this messages for the past few hours. I turned off snipe since last night.
(ATTENTION) No usable PokeStops found in your area. Is your maximum distance too small?

**Comment by zenwaichi** [comment]
@kivelaBE  there's any shortcut to pull this fork?

**Comment by Newgamer1** [comment]
@kivelaBE  how do i change it?

**Comment by zenwaichi** [comment]
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS1061	'ILogicSettings' does not contain a definition for 'TotalAmountOfPokebalsToKeep' and no extension method 'TotalAmountOfPokebalsToKeep' accepting a first argument of type 'ILogicSettings' could be found (are you missing a using directive or an assembly reference?)	PoGo.NecroBot.Logic	C:\Users\user\Desktop\git\NecroBot\PoGo.NecroBot.Logic\Inventory.cs	263	
Error	CS1061	'ILogicSettings' does not contain a definition for 'TotalAmountOfPotionsToKeep' and no extension method 'TotalAmountOfPotionsToKeep' accepting a first argument of type 'ILogicSettings' could be found (are you missing a using directive or an assembly reference?)	PoGo.NecroBot.Logic	C:\Users\user\Desktop\git\NecroBot\PoGo.NecroBot.Logic\Inventory.cs	278	
Error	CS1061	'ILogicSettings' does not contain a definition for 'TotalAmountOfRevivesToKeep' and no extension method 'TotalAmountOfRevivesToKeep' accepting a first argument of type 'ILogicSettings' could be found (are you missing a using directive or an assembly reference?)	PoGo.NecroBot.Logic	C:\Users\user\Desktop\git\NecroBot\PoGo.NecroBot.Logic\Inventory.cs	293	
Error	CS1061	'ILogicSettings' does not contain a definition for 'TransferConfigAndAuthOnUpdate' and no extension method 'TransferConfigAndAuthOnUpdate' accepting a first argument of type 'ILogicSettings' could be found (are you missing a using directive or an assembly reference?)	PoGo.NecroBot.Logic	C:\Users\user\Desktop\git\NecroBot\PoGo.NecroBot.Logic\State\VersionCheckState.cs	178	
Error		Metadata file 'C:\Users\user\Desktop\git\NecroBot\PoGo.NecroBot.Logic\bin\Debug\PoGo.NecroBot.Logic.dll' could not be found	PoGo.NecroBot.CLI	C:\Users\user\Desktop\git\NecroBot\PoGo.NecroBot.CLI\CSC		


get this on build (cloned Necronomicon's master and pulled your fork then ran git submodule update --init --recursive).

**Comment by kivelaBE** [comment]
will clone the master again, it probably had an update in the mean while.  Give me a few minutes

File: PoGo.NecroBot.Logic/ILogicSettings.cs [hunk 2]
```diff
@@ -96,11 +103,7 @@ public interface ILogicSettings
         int SnipeLocationServerPort { get; }
         bool UseSnipeLocationServer { get; }
         bool UseTransferIVForSnipe { get; }
-        bool SnipeIgnoreUnknownIV { get; }
         int MinDelayBetweenSnipes { get; }
-        int TotalAmountOfPokebalsToKeep { get; }
-        int TotalAmountOfPotionsToKeep { get; }
-        int TotalAmountOfRevivesToKeep { get; }
```
Comment (zenwaichi) [inline PoGo.NecroBot.Logic/ILogicSettings.cs]: Why those deletions? VS is complaining about the lack of those declarations

Comment (kivelaBE) [inline PoGo.NecroBot.Logic/ILogicSettings.cs]: I didn't, they got added to the master later than I pulled the master.  Am updating as we speak

Comment (kivelaBE) [inline PoGo.NecroBot.Logic/ILogicSettings.cs]: I updated my files, you should be able to compile them now

**Comment by zenwaichi** [comment]
restoring the missing getters from the interface led to successful build and later testing shower your pull merge to be working as intended. Well done

**Comment by l3uddz** [comment]
This would be a great addition to this project https://github.com/5andr0/PogoLocationFeeder

**Comment by NecronomiconCoding** [comment]
This branch has conflicts that must be resolved

**Comment by zenwaichi** [comment]
there are tons of useless and detrimental commits and accepted pr and when a good and promising one appears it gets the cold shoulder. Just wow

**Comment by kivelaBE** [comment]
Thanks @zenwaichi !

**Pull request closed without merging**